app/src/main/java/sdp/moneyrun/MenuActivity.java

```diff
@@ -199,16 +208,28 @@ public PopupWindow onButtonShowPopupWindowClick(View view, Boolean focusable, in
 
         return popupWindow;
     }
-
-
-
-//    @Override
-//    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
-//        switch (item.getItemId()){
-//            case R.id.profile_button:
-////                getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, new PlayerProfileFragment()).commit();
-//                break;
-//        }
-//        return false;
-//    }
+    //TODO: fix it somehow: task is never completed and thus cannot get player from database
+    public void setPlayerObject(){
```

Comment:
Same here, a big function that is not tested, would greatly improve the coverage if tested.

Reply:
On it